Remove-Duplicates-from-Sorted-Array

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Please insert in correct file structure with correct file names. Make a separate file for question and solution.